readme.md

```diff
@@ -380,6 +380,7 @@
 - [PowerShell](https://github.com/janikvonrotz/awesome-powershell#readme) - Cross-platform object-oriented shell.
 - [Alfred Workflows](https://github.com/derimagia/awesome-alfred-workflows#readme) - Productivity app for macOS.
 - [Terminals Are Sexy](https://github.com/k4m4/terminals-are-sexy#readme)
+- [Git Hooks](https://github.com/aitemr/awesome-git-hooks#readme)
```

Comment:
> The entry in the Awesome list should:
Include a short description about the project/theme of the list. It should not describe the list itself.
Example: - [Fish](…) - User-friendly shell., not - [Fish](…) - Resources for Fish..